app.py

```diff
@@ -211,16 +211,14 @@ def calc_data(pair, range=0.05, maxSize=32, minVolumePerc=0.01, ob_points=30):
         round_sig, args=(0,))
     final_tbl['sqrt'] = np.sqrt(final_tbl[TBL_VOLUME])
     final_tbl['total_price'] = (((final_tbl['volume'] * final_tbl['price']).round(2)).apply(lambda x: "{:,}".format(x)))
+    bid_tbl['total_price'] = bid_tbl['volume'] * bid_tbl['price']
+    ask_tbl['total_price'] = ask_tbl['volume'] * ask_tbl['price']
 
     # Get Dataset for Volume Grouping
-    vol_grp_bid = bid_tbl.groupby([TBL_VOLUME]).agg({TBL_PRICE: [np.min, np.max, 'count'], TBL_VOLUME: np.sum}).rename(
-        columns={'amin': 'min_Price', 'amax': 'max_Price', 'sum': TBL_VOLUME})
-    vol_grp_ask = ask_tbl.groupby([TBL_VOLUME]).agg({TBL_PRICE: [np.min, np.max, 'count'], TBL_VOLUME: np.sum}).rename(
-        columns={'amin': 'min_Price', 'amax': 'max_Price', 'sum': TBL_VOLUME})
-
-    # Get rid of header group row
-    vol_grp_bid.columns = vol_grp_bid.columns.droplevel(0)
-    vol_grp_ask.columns = vol_grp_ask.columns.droplevel(0)
+    vol_grp_bid = bid_tbl.groupby([TBL_VOLUME]).agg({TBL_PRICE: [np.min, np.max, 'count'], TBL_VOLUME: np.sum, 'total_price': np.sum})
+    vol_grp_bid.columns = ['min_Price', 'max_Price', 'count', 'volume', 'total_price']
```

Comment:
Please reorder it to met style of rest 

Comment
task on bid
task on ask

Comment
task bid
task ask

Reply:
Ok, done